arm_scenes.go

```diff
@@ -452,175 +451,3 @@ func scene9(ctx context.Context, logger logging.Logger) (*armplanning.PlanReques
 		WorldState:  worldState,
 	}, nil
 }
-
```

Comment:
I don't feel a need to remove these; they should run near-instantly so shouldn't negatively impact us to just stay there.

Reply:
Fair